src/Scanner.java

```diff
@@ -205,6 +207,16 @@
   }
 
   /**
+  * Specifies if the scan will be in reverse or not
+  * @param to_reverse Indication of scan direction. If this is not
+  * invoked, scanning will default to not being reversed
```

Comment:
Nit: Add a period at the end of the javadoc to match the rest of the code style.